docs/general/02_building-and-testing.md

```diff
@@ -18,12 +18,11 @@ The framework can be unit-tested by `cd`ing into `test` folder and calling
 dotnet msbuild /t:RunTests
 ```
 
-Moreover, there are some integration-style tests that require external services. They can be tested with `docker` and `docker-compose` tool. There are two integration-test suites:
+Moreover, there are some integration-style tests that require external services. They can be tested with `docker` and `docker-compose` tool. There is one integration-test suite currently:
 
- 1. `test/IntegrationTests` - _real_ integration tests,
- 2. `test/Testing/LeanCode.IntegrationTestHelpers.Tests` - test for the integration test helpers.
+ 1. `test/LeanCode.IntegrationTests` - _real_ integration tests,
```

Comment:
This clarification seems unnecessary now that there is only one test suite.

Also, what is `test/Testing/…` supposed to be now that it's not mentioned here anymore?

Reply:
`test/Testing` is no longer integration - it is run normally, along with other tests.